blogposts/2022/sourcegraph-non-devs.md

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Sourcegraph for non-devs'
+description: "Sourcegraph isn't just for software developers. Learn how product managers and analytics/business operations teammates use Sourcegraph as a powerful timesaver and cheat sheet that saves them 20-60 minutes multiple times per week."
```

Comment:
Again, I think we could give just a hint more about what people are saving time on/being able to self serve